CHANGELOG.md

```diff
@@ -23,7 +23,10 @@ To release a new version (e.g. from `1.0.0` -> `2.0.0`):
 
 ## [Unreleased]
 
-## [0.2.0] - 2022-01-06
+* Dropped support for Python 3.8, following [NEP 29].
+* Added experimental support for Python 3.12.0-rc.2.
```

Comment:
You can probably just say "Added support for Python 3.12", since upstream CPython encourages folks to ship wheels with the -rc releases.

Reply:
Done